From patchwork Mon Feb 19 20:37:24 2024
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
X-Patchwork-Id: 13563092
Return-Path: 
 <linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from bombadil.infradead.org (bombadil.infradead.org
 [198.137.202.133])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by smtp.lore.kernel.org (Postfix) with ESMTPS id 78A04C48BF8
	for <linux-rockchip@archiver.kernel.org>;
 Mon, 19 Feb 2024 20:37:43 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=bombadil.20210309; h=Sender:
	Content-Transfer-Encoding:Content-Type:List-Subscribe:List-Help:List-Post:
	List-Archive:List-Unsubscribe:List-Id:MIME-Version:Message-ID:Date:Subject:Cc
	:To:From:Reply-To:Content-ID:Content-Description:Resent-Date:Resent-From:
	Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:In-Reply-To:References:
	List-Owner; bh=96yNM7Teud8EbnJdYRFsDN3GsdGUZq0b3dFbgZGpLM4=; b=CDYt5HufjKni+z
	wr5LAYxdHONHJHpi0OVZqh9RQ0rRH9w8UifOExBYn0W3JvKlBBU0EM9Jp0IjiwhiVuMZTIi/tVrdd
	+wfWRLxiZY+EH/HJlQtcCVtVWh2x38qJ2zb0RP8w/Rn+NpaKO2YcfAcLOPHolwY5zX0LbXOsGDxAf
	atbqIrxx2XkAIfrm2QrdkXydFZTo6iJgLqdNNuMbFhe1PIpLtuOkQ5GeUQz5HaVbDdlCXX8LokibY
	c+3TxmJUKYXkSsXiJa47Mg/Mc5Wc/iWSvebwVxjxm0ckl9pybGtTHjKhCXtgGddfdFyJzR9PayRt6
	fHhXEyZ2e3HNK8pNpqKg==;
Received: from localhost ([::1] helo=bombadil.infradead.org)
	by bombadil.infradead.org with esmtp (Exim 4.97.1 #2 (Red Hat Linux))
	id 1rcAOP-0000000C7Lu-0BbU;
	Mon, 19 Feb 2024 20:37:37 +0000
Received: from madrid.collaboradmins.com ([46.235.227.194])
	by bombadil.infradead.org with esmtps (Exim 4.97.1 #2 (Red Hat Linux))
	id 1rcAOL-0000000C7Ke-1VdN;
	Mon, 19 Feb 2024 20:37:34 +0000
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=collabora.com;
	s=mail; t=1708375047;
	bh=tmU1TAOSo2/1xTTWVq6GYLkUWbf4ZZznDP5eiUCLTNE=;
	h=From:To:Cc:Subject:Date:From;
	b=hkAu+vXwKAEuAMy5kLMyDoyXdErhbrKZ9+kGwesEArRBbCHvXVjTmc0Xd4lJd8Ob2
	 X0eiapSQlw8Qt5aTTi9xwFPiLTlXPI5WB7hTcI59kIn9zSXLX3cF4iAtx3MkeBQLmm
	 SKUDG6CZjTn5+JPnbcnMYznQdTjlPLaWmUVCaq0WHFKFPm5pgNjQZ83vHERTU3gI30
	 2e3sGXoD9mNH8UrnjANc5ZQqbhneEQ3mW1nlSfcNogIjyFussHMLyIaEdaOSTNzqL4
	 x9iy0qmpuoYM+XqigIBENMlKs1TmlJzsDFP6tBcZnkbTMKpnNoyftqQqAzE2cIPFlK
	 5Ec74kFtAMHiQ==
Received: from localhost (cola.collaboradmins.com [195.201.22.229])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange ECDHE (prime256v1) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
	(No client certificate requested)
	(Authenticated sender: cristicc)
	by madrid.collaboradmins.com (Postfix) with ESMTPSA id 33FFA3782080;
	Mon, 19 Feb 2024 20:37:27 +0000 (UTC)
From: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
To: =?utf-8?q?Heiko_St=C3=BCbner?= <heiko@sntech.de>,
 Catalin Marinas <catalin.marinas@arm.com>, Will Deacon <will@kernel.org>
Cc: linux-arm-kernel@lists.infradead.org,
	linux-kernel@vger.kernel.org,
	linux-rockchip@lists.infradead.org,
	kernel@collabora.com
Subject: [PATCH] arm64: defconfig: Enable Rockchip HDMI/eDP Combo PHY
Date: Mon, 19 Feb 2024 22:37:24 +0200
Message-ID: <20240219203725.283532-1-cristian.ciocaltea@collabora.com>
X-Mailer: git-send-email 2.43.2
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20240219_123733_549447_9C4AEDBF 
X-CRM114-Status: UNSURE (   8.16  )
X-CRM114-Notice: Please train this message.
X-BeenThere: linux-rockchip@lists.infradead.org
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: Upstream kernel work for Rockchip platforms
 <linux-rockchip.lists.infradead.org>
List-Unsubscribe: <http://lists.infradead.org/mailman/options/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-rockchip/>
List-Post: <mailto:linux-rockchip@lists.infradead.org>
List-Help: <mailto:linux-rockchip-request@lists.infradead.org?subject=help>
List-Subscribe: <http://lists.infradead.org/mailman/listinfo/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=subscribe>
Sender: "Linux-rockchip" <linux-rockchip-bounces@lists.infradead.org>
Errors-To: 
 linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org

Enable support for the Rockchip HDMI/eDP Combo PHY, which is based on a
Samsung IP block. This is used by the RK3588 SoC family.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 056a6cc546a4..b1f8ff32f940 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -1510,6 +1510,7 @@ CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_PHY_ROCKCHIP_INNO_DSIDPHY=m
 CONFIG_PHY_ROCKCHIP_NANENG_COMBO_PHY=m
 CONFIG_PHY_ROCKCHIP_PCIE=m
+CONFIG_PHY_ROCKCHIP_SAMSUNG_HDPTX=m
 CONFIG_PHY_ROCKCHIP_SNPS_PCIE3=y
 CONFIG_PHY_ROCKCHIP_TYPEC=y
 CONFIG_PHY_SAMSUNG_UFS=y
